src/common/colorspaces.c

```diff
@@ -338,9 +351,17 @@ static cmsHPROFILE dt_colorspaces_create_gamma_rec709_rgb_profile(void)
 {
   cmsFloat64Number srgb_parameters[5] = { 2.2, 1.0 / 1.099,  0.099 / 1.099, 1.0 / 4.5, 0.081 };
   cmsToneCurve *transferFunction = cmsBuildParametricToneCurve(NULL, 4, srgb_parameters);
+  cmsCIExyYTRIPLE primaries;
+
+  cmsXYZ2xyY(&primaries.Red,   &rec709_primaries_prequantized.Red);
+  cmsXYZ2xyY(&primaries.Green, &rec709_primaries_prequantized.Green);
+  cmsXYZ2xyY(&primaries.Blue,  &rec709_primaries_prequantized.Blue);
+
+  cmsCIExyY d65xyY;
+  cmsXYZ2xyY(&d65xyY, &d65);
 
   cmsHPROFILE profile = _create_lcms_profile("Gamma Rec709 RGB", "Gamma Rec709 RGB",
-                                             &rec709_primaries_pre_quantized, transferFunction, TRUE);
+                                             &d65xyY, &primaries, transferFunction, TRUE);
```

Comment:
same about the D50 to D65 change

Reply:
All previous primaries where D65 (at least that's what previous implementation was using when creating the profile). So I kept D65, but there is zero documentation about that.

For sure the new ProPhoto I have implemented has primaries for D50.

Reply:
The previous code was doing:

```
  cmsWriteTag(profile, cmsSigMediaWhitePointTag, &d65);
  cmsWriteTag(profile, cmsSigMediaBlackPointTag, &black);
```

Reply:
Unconditionally for all created profiles.

Reply:
Ok if the previous code used D65. You just need to check that the primaries are valid for the white point that is tagged.